src/main/java/uk/gov/hmcts/reform/civil/service/robotics/mapper/EventHistoryMapper.java

```diff
@@ -707,12 +707,16 @@ private void buildConsentExtensionFilingDefence(EventHistory.EventHistoryBuilder
                     .eventCode(CONSENT_EXTENSION_FILING_DEFENCE.getCode())
                     .dateReceived(dateReceived)
                     .litigiousPartyID("002")
-                    .eventDetailsText(format("agreedExtensionDate: %s", extensionDate
-                        .format(ISO_DATE)))
-                    .eventDetails(EventDetails.builder()
-                                      .agreedExtensionDate(extensionDate
-                                                               .format(ISO_DATE))
-                                      .build())
+                    .eventDetailsText(
```

Comment:
This part of code has been also changed in CMC-1679 as per RPA event text guidance Confluence page. If that PR-700 (CMC-1679 branch) is good, then we probably don't need this overlapping PR. This PR also seems to be containing stuff from other PRs not yet merged in master.

Reply:
I realise the date format change also contained here. We can carry this particular change on top of 1679 PR or merge this PR after 1679 is merged, if you agree